packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -123,4 +133,8 @@ export default class QuanticSearchInterface extends LightningElement {
   onHashChange = () => {
     this.urlManager.synchronize(this.fragment);
   };
+
+  get contentCssClasses() {
+    return !this.isInitialized ? 'search__content_hidden' : '';
```

Comment:
```suggestion
    return !this.isInitialized && 'search__content_hidden' || '';
```
Do you like this format? Just trying stuff out

Reply:
I do prefer the ternary in this case.

Reply:
Ternary are more explicit imo.

Reply:
👍